builtin/providers/aws/resource_aws_cloudtrail.go

```diff
@@ -84,6 +89,11 @@ func resourceAwsCloudTrailCreate(d *schema.ResourceData, meta interface{}) error
 
 	d.SetId(*t.Name)
 
+	// AWS CloudTrail sets newly-created trails to false.
+	if v, ok := d.GetOk("enable_logging"); ok && v.(bool) {
+		cloudTrailSetLogging(conn, v.(bool), d.Id())
```

Comment:
Can you add error checking here, please? Otherwise any error coming out from `cloudTrailSetLogging` would be ignored.